nhaystack-rt/src/nhaystack/util/TypeUtil.java

```diff
@@ -226,6 +228,46 @@ else if (def instanceof BSimple)
 
             return simple;
         }
+        else if (def instanceof BEnumOverride)
+        {
+            BEnumOverride cpx = (BEnumOverride) def;
+            BFacets facets = comp.getAction(action.getName()).getFacets();
+            BEnumRange range = (BEnumRange) facets.get(BFacets.RANGE);
+
+            Iterator<Map.Entry<String, HVal>> it = args.iterator();
+            while (it.hasNext())

```

Comment:
What is the reason to loop here?  Are you expecting more than 1 argument to be present?

Reply:
Yes, duration and maxOverrideDuration